src/loot_mover.py

```diff
@@ -0,0 +1,92 @@
+from src.cam import Cam
+from src.config.loader import IniConfigLoader
+from src.config.models import MoveItemsType
+from src.logger import Logger
+from src.ui.char_inventory import CharInventory
+from src.ui.chest import Chest
+from src.utils.custom_mouse import mouse
+
+
+def move_items_to_stash():
+    Logger().info("Move inventory items to stash")
+
+    inv = CharInventory()
+    chest = Chest()
+
+    if chest.is_open():
+        occupied_inv, _ = inv.get_item_slots()
+
+        for i in IniConfigLoader().general.check_chest_tabs:
+            chest.switch_to_tab(i)
+
+            _, empty_chest = chest.get_item_slots()
+
+            if len(empty_chest) == 0:
+                continue
+
+            move_items(inv, occupied_inv, len(empty_chest))
+
+            if len(occupied_inv) == 0:
+                break
+
+        mouse.move(*Cam().abs_window_to_monitor((0, 0)))
+        Logger().info("Completed move")
+    else:
+        Logger().error("Moving items only works if stash is open")
+
+
+def move_items_to_inventory():
+    Logger().info("Move stash items to inventory")
+
+    inv = CharInventory()
+    chest = Chest()
+
+    if chest.is_open():
+        _, empty_inv = inv.get_item_slots()
+        empty_slot_count = len(empty_inv)
+
+        for i in IniConfigLoader().general.check_chest_tabs:
+            chest.switch_to_tab(i)
+
+            occupied_chest, _ = chest.get_item_slots()
+
+            Logger().debug(
+                f"Stash tab {i} - Number of stash items: {len(occupied_chest)} - Number of empty inventory " f"spaces: {empty_slot_count}"
+            )
+
+            item_move_count = move_items(inv, occupied_chest, empty_slot_count)
+            empty_slot_count = empty_slot_count - item_move_count
+            Logger().debug(f"Moved {item_move_count} items, now have {empty_slot_count} empty slots left.")
+
+            if empty_slot_count < 1:
+                break
+
+        mouse.move(*Cam().abs_window_to_monitor((0, 0)))
+        Logger().info("Completed move")
+    else:
+        Logger().error("Moving items only works if stash is open")
+
+
+def move_items(inv, occupied, num_to_move):
```

Comment:
I don't know if this was intentional or not, but some types are passed by reference in Python, and others are not. In your case, you are changing the list of occupied. This makes it very hard to understand when logic gets more complicated. I would recommend refactoring this to avoid doing that.

Reply:
It was intentional to modify occupied, but I can have it return a tuple instead if that makes more sense.

Reply:
I've made the change I mentioned above, let me know if that's what you were thinking.

Reply:
Nice!